.github/workflows/build.yml

```
@@ -55,7 +55,63 @@ jobs:
             ${{ steps.get_info.outputs.file_prefix }}-right.uf2
+  build-clique:
+    runs-on: ubuntu-latest
+    container:
+      image: zmkfirmware/zmk-build-arm:stable
+    name: Build (Clique)
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Get version data
+        id: get_info
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:5:36: Double quote to prevent globbing and word splitting

(shellcheck)